### ISSUE TITLE
Make Voyager use Explorer's page for displaying validator information.

### ISSUE THREAD

**Issue by NodeGuy** [cosmos/voyager#1154]
This is an intermediate step towards https://github.com/cosmos/voyager/issues/1100.

**Comment by faboweb** [comment]
Probably moving the description properties to the top level of the object. I guess our internal delegate object didn't match the external object anymore. Feel free to change that, if it's not too much effort.

**Comment by faboweb** [comment]
What is this blocked on?

**Comment by fedekunze** [comment]
Let's wait for @jbibla new designs and use them instead

**Comment by faboweb** [comment]
We can still migrate to the new designs later. The new design is not part of this ticket. Please implement the explorer page as is if you need that to finish #1100

**Comment by NodeGuy** [comment]
I marked it `blocked` as a note to myself to show that I was waiting for your answer.

**Comment by NodeGuy** [comment]
> Probably moving the description properties to the top level of the object. I guess our internal delegate object didn't match the external object anymore. Feel free to change that, if it's not too much effort.

Thanks.  It's causing your PR to display the moniker incorrectly so I tried removing it but doing so broke the validator list and I'm having trouble figuring out why.
